operators/src/engine/query.rs

```diff
@@ -43,37 +44,51 @@ impl TestDefault for ChunkByteSize {
 pub trait QueryContext: Send + Sync {
     fn chunk_byte_size(&self) -> ChunkByteSize;
     fn thread_pool(&self) -> &Arc<ThreadPool>;
+
+    fn valve(&self) -> &Valve;
+    fn valve_trigger(&mut self) -> Option<Trigger>;
```

Comment:
especially here

Reply:
I changed the name to abort_registration and abort_trigger https://github.com/geo-engine/geoengine/commit/db426f1844ce4883ad3021252bbf6bbfee69eeb5